docs/_content/standaard/standaard.md

```diff
@@ -235,6 +235,12 @@ controleren van alle gerelateerde informatieobjecten. Deze MAG NIET `null` zijn,
 maar MOET `true` of `false` zijn. Indien dit niet het geval is, dan dient het
 ZRC een validatiefout te genereren met code `indicatiegebruiksrecht-unset`.
 
+#### Heropenen zaak
+
+Bij het heropenen van een zaak MOET de client een andere status toevoegen aan de zaak dan een eindstatus. 
+Tevens MOET de client de 'Zaak.archiefactiedatum' leeg maken en het resultaat verwijderen. Het verwijderen van een resultaat MOET in de audit trail zichtbaar zijn.
```

Comment:
Is het niet aan de client om te bepalen of het resultaat wel of niet leeggemaakt hoeft te worden?
```suggestion
Tevens MAG de client de 'Zaak.archiefactiedatum' leeg maken en het resultaat verwijderen. Het verwijderen van een resultaat MOET in de audit trail zichtbaar zijn.
```

Ik denk dat er ook nog wat meer afhankelijkheden zijn dan:

* geen resultaat -> archiefactiedatum MOET leeg zijn
* wel resultaat -> archiefactiedatum MAG leeg zijn (indien leeg gelaten, weer afleiden uit resultaattype)
* resultaat laten staan, archiefactiedatum leegmaken -> provider MOET datum opnieuw berekenen? wat dan met de gevallen waarin het niet kan berekend worden?
* wel archiefactiedatum, maar geen resultaat -> dat mag dus niet, lijkt me?

@joeribekker wat denk jij hiervan?

Reply:
> Is het niet aan de client om te bepalen of het resultaat wel of niet leeggemaakt hoeft te worden?

Eens, echter, ik zou neerzetten dat je `Resultaat` MAG verwijderen. Dat de `Zaak.archiefactiedatum` dan ook leeg wordt is gevolg van en moet niet expliciet op leeg gezet worden.

> Ik denk dat er ook nog wat meer afhankelijkheden zijn dan:
> geen resultaat -> archiefactiedatum MOET leeg zijn

Eens.

> wel resultaat -> archiefactiedatum MAG leeg zijn (indien leeg gelaten, weer afleiden uit resultaattype)

Ik zou zeggen: MOET afleiden bij het zetten/wijzigen van een `Resultaat` indien mogelijk. Je mag em in principe NIET zelf zetten, behalve als hij niet afgeleid kan worden.

> resultaat laten staan, archiefactiedatum leegmaken -> provider MOET datum opnieuw berekenen? wat dan met de gevallen waarin het niet kan berekend worden?

Leeg maken moet niet mogen denk ik.

> wel archiefactiedatum, maar geen resultaat -> dat mag dus niet, lijkt me?

Dit is een situatie die niet voor kan komen, zie eerste opmerking.